packages/transformer/src/test/TestUtils/TimelineTestUtil.ts

```diff
@@ -449,6 +449,9 @@ export async function runTimeline(
           wasSourceIModelCopiedToTarget: true,
         });
         await provenanceInserter.process();
+        provenanceInserter.updateSynchronizationVersion({
```

Comment:
How come you had to add this?

Reply:
Because this used to have the `_isProvenanceInitTransform` functionality that's why I added it there to save the first changeset.

Reply:
But shouldnt that occur by passing the truth value of isProvenancwInitTrnasfomr to updateSyncVersion at the end of process?